deploy/lib/server_config.rb

```diff
@@ -104,6 +104,9 @@ def initialize(options)
     else
       @qconsole_port = @bootstrap_port
     end
+
+    r = execute_query %Q{xdmp:host-name()}
+    @properties["ml.server-name"] = parse_body(r.body)
```

Comment:
Seems like this would also have the side effect of not allowing people to hit their load balancer? Couldn't that potentially be unwanted behavior for tools like MLCP?